main.tf

```diff
@@ -67,7 +67,7 @@ locals {
  * Create role for scheduled running of rclone task definition.
  */
 resource "aws_iam_role" "rclone_event" {
-  name = "rclone_event-${local.app_name_and_env}-s3copy"
+  name = "rclone_event-${local.app_name_and_env}-s3copy-${var.s3_bucket_name}"
```

Comment:
This may exceed the limits for an IAM role name, which I think is 64 characters.

Reply:
Good catch!  This [document](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html#reference_iam-quotas-entity-length) confirms the limit is 64 characters.  I think the longest name we currently are using is 54 characters.  Should we replace the bucket name with a six character random string?  Should we document the length restriction in the README.md file?

Reply:
I'll leave it up to you, but since it's already working and not truncated, I'd vote for not changing anything. Since you didn't have to specifically use short names to avoid the limit, it may be uncommon to have the problem in the future.